pkgs/by-name/do/dotcopy/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildGoModule
+, fetchzip
+}:
+
+buildGoModule rec {
+  pname = "dotcopy";
+  version = "0.2.14";
+
+  src = fetchzip {
```

Comment:
```suggestion
  src = fetchFromGitHub {
```

Reply:
I was originally using fetchFromGitHub and it wasn't working. Details of my suffering are here: [https://discourse.nixos.org/t/buildgomodule-fails-due-to-invalid-go-version/40823](https://discourse.nixos.org/t/buildgomodule-fails-due-to-invalid-go-version/40823)

Reply:
Try again.